benchmark/comparisonTest/main.go

```diff
@@ -17,31 +17,52 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"os"
-	"strconv"
 	"testing"
 
 	"github.com/awslabs/soci-snapshotter/benchmark"
 	"github.com/awslabs/soci-snapshotter/benchmark/framework"
 )
 
 var (
-	outputDir = "./output"
+	outputDir = "../comparisonTest/output"
 )
 
 func main() {
-	commit := os.Args[1]
-	configCsv := os.Args[2]
-	numberOfTests, err := strconv.Atoi(os.Args[3])
-	if err != nil {
-		errMsg := fmt.Sprintf("Failed to parse number of test %s with error:%v\n", os.Args[3], err)
-		panic(errMsg)
+
+	var (
+		numberOfTests           int
+		configCsv               string
+		showCom                 bool
+		parseFileAccessPatterns bool
+		imageList               []benchmark.ImageDescriptor
+		err                     error
+		commit                  string
+	)
+
+	flag.BoolVar(&parseFileAccessPatterns, "parseFileAccess", false, "Parse fuse file access patterns.")
```

Comment:
to match other flag formats:

```suggestion
	flag.BoolVar(&parseFileAccessPatterns, "parse-file-access", false, "Parse fuse file access patterns.")
```

Reply:
Updated.